### PR TITLE
[DOC] d2iq-96652 metallb considerations pushed for charles du

### PR DESCRIPTION
**Pull request by JohnB64-JBV**
## Jira Ticket

[](https://d2iq.atlassian.net/browse/D2IQ-96652)

## Description of changes being made
Created new page for MetalLB considerations during migration from 1.8.x to 2.1.x. (Baker pushed this for Charles Du who owns the Jira ticket)

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-<add_pr_##_here>.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by charIesdu — APPROVED** [review]

File: pages/dkp/konvoy/2.1/major-version-upgrade/migration-for-metallb/index.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+layout: layout.pug
+navigationTitle: Migration Considerations for MetalLB
+title: Migration Considerations for MetalLB
+menuWeight: 70
+excerpt: Some important considerations for MetalLB when when migrating from DKP 1.8.x to 2.1.x
+beta: true
+enterprise: false
+---
+
+The MetalLB project has moved all Docker images from dockerhub.io to quay.io.  This prevents older versions of DKP (up to and including DKP 2.1.x) from installing MetalLB correctly.     Attempting to migrate any cluster using MetalLB from Konvoy 1.8 to DKP 2.1 will fail due to this issue.
```
Comment (cmastr) [inline pages/dkp/konvoy/2.1/major-version-upgrade/migration-for-metallb/index.md:11]:
```suggestion
The MetalLB project has moved all Docker images from dockerhub.io to quay.io. This prevents older versions of DKP (up to and including DKP 2.1.x) from installing MetalLB correctly. Attempting to migrate any cluster using MetalLB from Konvoy 1.8 to DKP 2.1 will fail due to this issue. 
```

File: pages/dkp/konvoy/2.1/major-version-upgrade/migration-for-metallb/index.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+excerpt: Some important considerations for MetalLB when when migrating from DKP 1.8.x to 2.1.x
+beta: true
+enterprise: false
+---
+
+The MetalLB project has moved all Docker images from dockerhub.io to quay.io.  This prevents older versions of DKP (up to and including DKP 2.1.x) from installing MetalLB correctly.     Attempting to migrate any cluster using MetalLB from Konvoy 1.8 to DKP 2.1 will fail due to this issue. 
+
+Follow these steps to resolve a failed migration:
+
+1.  Retrieve the metallb-overrrides config map using the following command:
```
Comment (cmastr) [inline pages/dkp/konvoy/2.1/major-version-upgrade/migration-for-metallb/index.md:15]:
```suggestion
1.  Retrieve the metallb-overrrides configMap using the following command:
```

File: pages/dkp/konvoy/2.1/major-version-upgrade/migration-for-metallb/index.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+    ``` bash
+    kubectl edit configmap metallb-overrides -n kommander
```
Comment (cmastr) [inline pages/dkp/konvoy/2.1/major-version-upgrade/migration-for-metallb/index.md:18]:
```suggestion
    kubectl edit configMap metallb-overrides -n kommander
```

**Comment by msdolbey** [comment]
Hey @JohnB64-JBV @charIesdu - I made some linting/markdown fixes - other than that, this looks good.
Oh, also the yaml somehow got a little wacky, so I cleaned that up, too.

**Review by msdolbey — APPROVED** [review]
👍 
the real metallb

**Review by cmastr — APPROVED** [review]

**Review by cneth — APPROVED** [review]

**Pull request merged**